pkg/kubectl/cmd/get.go

```diff
@@ -126,7 +126,7 @@ func NewCmdGet(f cmdutil.Factory, out io.Writer, errOut io.Writer) *cobra.Comman
 		ArgAliases: argAliases,
 	}
 	cmdutil.AddPrinterFlags(cmd)
-	cmd.Flags().StringP("selector", "l", "", "Selector (label query) to filter on, supports '=', '==', and '!='.")
+	cmd.Flags().StringSliceP("selector", "l", []string{}, "Selector (label query) to filter on, supports '=', '==', and '!='. You can also use multiple flag options like -l label1=label1 -l label2=label2, but it will be aggregated to a comma separated selector")
```

Comment:
I think there's more selector flags not only in `kubectl get`, could they also be supported this feature ?